### ISSUE TITLE
Huge amount of messages in homebridge logging

### ISSUE THREAD

**Issue by CM-UH** [samemory/homebridge-eufy-security#94]
This plugin produces a lot of messages in the hombridge log-file.
This message comes every minute.

**Logs:**
`(node:4974) UnhandledPromiseRejectionWarning: TypeError: this.eufyDevice.isEnabled is not a function
    at CameraAccessory.handleOnGet (/usr/local/lib/node_modules/homebridge-eufy-security/src/accessories/CameraAccessory.ts:209:42)
    at On.emit (events.js:387:35)
    at /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:1508:14
    at new Promise (<anonymous>)
    at On.<anonymous> (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:1506:12)
    at step (/usr/local/lib/node_modules/homebridge/node_modules/tslib/tslib.js:143:27)
    at Object.next (/usr/local/lib/node_modules/homebridge/node_modules/tslib/tslib.js:124:57)
    at /usr/local/lib/node_modules/homebridge/node_modules/tslib/tslib.js:117:75
    at new Promise (<anonymous>)
    at Object.__awaiter (/usr/local/lib/node_modules/homebridge/node_modules/tslib/tslib.js:113:16)
(node:4974) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 186)
(node:4974) UnhandledPromiseRejectionWarning: TypeError: this.eufyDevice.isMotionDetected is not a function
    at CameraAccessory.isMotionDetected (/usr/local/lib/node_modules/homebridge-eufy-security/src/accessories/CameraAccessory.ts:178:46)
    at CameraAccessory.handleMotionDetectedGet (/usr/local/lib/node_modules/homebridge-eufy-security/src/accessories/CameraAccessory.ts:188:37)
    at MotionDetected.emit (events.js:387:35)
    at /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:1508:14
    at new Promise (<anonymous>)
    at MotionDetected.<anonymous> (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:1506:12)
    at step (/usr/local/lib/node_modules/homebridge/node_modules/tslib/tslib.js:143:27)
    at Object.next (/usr/local/lib/node_modules/homebridge/node_modules/tslib/tslib.js:124:57)
    at /usr/local/lib/node_modules/homebridge/node_modules/tslib/tslib.js:117:75
    at new Promise (<anonymous>)
(node:4974) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 188)`
```

**Plugin Config:**
` {
            "username": "xxx",
            "password": "xxx",
            "hkHome": 1,
            "hkAway": 0,
            "hkNight": 3,
            "hkOff": 63,
            "hkDisarmed": 6,
            "platform": "EufySecurity"
        }`


**Environment:**

* **Plugin Version**: v0.2.12
* **Homebridge Version**: v1.3.4
* **Node.js Version**: v14.17.3
* **NPM Version**: v7.20.1
* **Operating System**: <!-- Raspbian 5.10.17-v7l+

**Comment by schliemann** [comment]
Duplicate of https://github.com/samemory/homebridge-eufy-security/issues/92

**Comment by lenoxys** [comment]
Should be fixed #93
